src/Editor/index.js

```diff
@@ -904,6 +904,7 @@ export class Editor extends React.Component {
             className={"ve-clickable ve-close-panel-button"}
             style={{
               zIndex: 15001,
+              minWidth: "200px !important",
```

Comment:
@juFelipe what's this fixing here? Seems like it would make the close button unnecessarily large?

Reply:
oh I was trying to fix another bug, looking for a solution and I forget to delete that, I'm sorry

Reply:
no worries. change it and I'll merge it in and publish :)